snapcraft/plugins/python.py

```diff
@@ -327,7 +328,8 @@ def _install_project(self):
                 # There is also a chance that this setup.py is distutils based
                 # in which case we will rely on the `pip install .` ran before
                 #  this.
-                with contextlib.suppress(subprocess.CalledProcessError):
+                with contextlib.suppress(subprocess.CalledProcessError,
```

Comment:
Oh good catch, we would have had issues! However, given that this uses `self.run`, there's no longer a need to catch `CalledProcessError` here, is there?